content/introduction/how-ipfs-works.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How IPFS Works
+weight: 2
+---
+
+IPFS is a peer-to-peer (p2p) network that links content-addressed data to other content-addressed data. Content is accessible through peers that might relay information or store it (or do both!), and those peers can be located anywhere in the world. IPFS knows how to find what you ask for by its content address, rather than where it is.
+
+<!-- <img alt="IPFS Stack" src="../assets/ipfs_stack.png" width="200px" /> -->
```

Comment:
@meiqimichelle Are the images going to be uncommented?

Reply:
They didn't seem to add much, I think, in the end. However! I should delete them, and not leave them commented out.

Reply:
Removed with https://github.com/ipfs/docs/pull/170/commits/2aa10f5ca047a43bf393a92696704f1695a83d80